spec/spec_helper.rb

```diff
@@ -8,10 +8,19 @@
 
 require "manageiq-providers-ibm_power_vc"
 
+def sanitizer(interaction)
+  # Remove internal network hostnames
+  interaction.response.body.gsub!(/"host": ".*?"/, '"host": "powervc.local"')
```

Comment:
I'm worried this might be too aggressive (even with the `?` operator to make it not greedy).  Perhaps:

```suggestion
  interaction.response.body.gsub!(/"host": "[^"]*?"/, '"host": "powervc.local"')
```